wts/resources/oauth2.py

```diff
@@ -23,7 +23,7 @@ def client_do_authorize():
         raise AuthError("could not authorize; state did not match across auth requests")
     try:
         tokens = client.fetch_token(token_url, **flask.request.args.to_dict())
-        return refresh_refresh_token(tokens, requested_idp)
```

Comment:
what's the reason for removing the return here? Just b/c it wasn't used and to clean this up?

Reply:
this was changed in #40, and yes iirc it was to clean up
https://github.com/uc-cdis/workspace-token-service/pull/36#discussion_r937016968